app/views/components/blockEditor.html

```diff
@@ -128,10 +128,22 @@
           <translate>Rules can depend on questions of</translate> <a href="" tooltip="{{'Multiple Choice, Dropdown, Number, Date, Gender or Year in School' | translate}}" translate>these types</a> <translate>that are above this question.</translate></p>
 
         <div class="form-horizontal">
-          <div class="form-group">
-            <div class="col-md-12">
+           <div class="form-group">
+            <div class="col-md-2">
               <button class="btn btn-success" ng-click="addRule()"><i class="fa fa-plus"></i> <translate>Add Rule</translate></button>
             </div>
```

Comment:
Needs `translate`. Also I'm not sure `operand` is a word an average user would know.

Maybe `Show question if: (·) All rules are met ( ) Any rules are met`